docker/nginx-default.conf

```diff
@@ -1,16 +1,30 @@
 server {
     listen 3001;
 
+    # External API
     location / {
         include cors.conf;
         proxy_pass http://node1:3013/;
     }
 
+    # Internal API
     location /internal/ {
+        # Backward compatibility
         include cors.conf;
         proxy_pass http://node1:3113/;
     }
 
+    location /v2/debug {
+        include cors.conf;
+        proxy_pass http://node1:3113/v2/debug;
```

Comment:
Why URI specified? Not clear whether it has intention.

Reply:
Because nginx pass the path without the prefix only.

Reply:
Interesting.

From [the manual](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_pass):
> If proxy_pass is specified without a URI, the request URI is passed to the server in the same form as sent by a client when the original request is processed, or the full normalized request URI is passed when processing the changed URI:
```
location /some/path/ {
    proxy_pass http://127.0.0.1;
}
```

Reply:
Perhaps you're reading latest docs. The version used in this setup is 2 years old

Reply:
I tested it locally and it works. Applied in 878de61b

I think the previous proxy_pass had `/` in the argument, thus interpreted as path